src/main.rs

```diff
@@ -24,25 +31,34 @@ fn time_to_mine(block_height: u64) -> Duration {
     // when the lazy macro is expanded
     // if a value has a static lifetime then it means that value lives as long as the program lives
     let rpc_client: &Client = &*RPC_CLIENT;
-    rpc_client.get_block_hash(234);
-    todo!()
+    let current_block: Block = get_block(block_height, rpc_client).unwrap();
+    let next_block: Block = get_block(block_height + 1, rpc_client).unwrap();
+
+    let current_block_time = current_block.header.time;
+    let next_block_time = next_block.header.time;
+
+    let time_to_mine_current_block = next_block_time - current_block_time;
+    Duration::new(time_to_mine_current_block.into(), 1).unwrap()
 }
 
 // TODO: Task 2
 fn number_of_transactions(block_height: u64) -> u16 {
-    let some_value = Box::new(4 as u32);
-    todo!()
+    let rpc_client: &Client = &*RPC_CLIENT;
+    let block = get_block(block_height, rpc_client).unwrap();
```

Comment:
Consider returning and handling an error with the use of the `?` operator instead of `unwrap`

Reply:
i have fixed this